### DISCUSSION TITLE
If the default locale is inside the url, the locale is not maintained during routing

### DISCUSSION THREAD

**Discussion by KoYoungWock** [i18next/next-i18next#2282] [Q&A]
```
"next": "^13.4.2",
"next-i18next": "^15.1.1",
"i18next": "^23.7.11",
```

```
/** @type {import('next-i18next').UserConfig} */

module.exports = {
  i18n: {
    defaultLocale: 'ko',
    locales: ['en', 'ko'],
    localeDetection: false,
  },
}
```

```ts
import { i18n } from 'next-i18next'

const locale = i18n?.language  ??' ko'

const handleClick = useCallback(
  () => {
    push(ROUTE_PATH.DETAIL, undefined, { locale )
  },
  [push],
)
```

Situation: When I launch the web in the app, there is a locale in the url, but when I push like above on the web, there is a problem that the locale is not maintained in the url.

After looking into it, I saw the next-i18next document saying that if defaultLocale's locale is in url, erasing locale in url, I'd appreciate it if you could let me know if there's anything I missed.

**Comment by adrai** [comment]
I don't know if I understood correctly, but try to push it like this: https://github.com/i18next/next-i18next/blob/master/examples/simple/pages/index.tsx#L24
